ably/rest/rest.py

```diff
@@ -26,11 +29,11 @@ def __init__(self, options=None):
           - `key`: a valid key string
 
           **Or**
-          - `key_id`: Your Ably key id
-          - `key_value`: Your Ably key value
+          - `keyId`: Your Ably key id
```

Comment:
We can just remove keyId and keyValue as documented options, they are not used.  clientId description can be found in other client libs, such as https://github.com/ably/ably-js/blob/master/common/lib/client/auth.js#L117